sql_user_check_snowflake/run.sh

```diff
@@ -8,13 +8,27 @@ readonly USERNAME=$SF_USERNAME
 readonly DATABASE=$SF_DATABASE
 readonly WAREHOUSE=$WAREHOUSE_NAME
 readonly ROLE=$ROLE_NAME
+readonly MODE=$MODE
 
 if ! command -v "$CMD" &> /dev/null
 then
     echo "Command $CMD could not be found, Follow https://docs.snowflake.com/en/user-guide/snowsql-install-config to install $CMD"
     exit 1
 fi
 
-tempQueryFile=$(mktemp)
-sed "s/LB_DATABASE_NAME/$DATABASE/g" queries.sql > "$tempQueryFile"
-snowsql -a "$ACCOUNT" --username "$USERNAME" -f "$tempQueryFile" -w "$WAREHOUSE" -r "$ROLE"
+
+if [ "$MODE" == "stats" ]; then
+  tempQueryFile=$(mktemp)
+  sed "s/LB_DATABASE_NAME/$DATABASE/g" stats_queries.sql > "$tempQueryFile"
+  snowsql -a "$ACCOUNT" --username "$USERNAME" -f "$tempQueryFile" -w "$WAREHOUSE" -r "$ROLE"
```

Comment:
Save the output to a file and ask user for output path

Reply:
done